src/ParaTest/Runners/PHPUnit/SuiteLoader.php

```diff
@@ -230,7 +236,7 @@ private function getMethodTests($class, $method, $useDataProvider = false)
 
     private function testMatchGroupOptions($groups)
     {
-        if (empty($groups)) {
+        if (empty($this->options->groups) && empty($this->options->excludeGroups) && empty($groups)) {
```

Comment:
This breaks for test methods without annotation inside a test class with one.
```php
/**
 * @group foo
 */
class TestClass
{
    public function testMethod() {}
}
```
It works on `initSuites` but then it will fail for `getMethodTests`